lesson07.md

```diff
@@ -0,0 +1,363 @@
+<!-- .slide: id="lesson7" -->
+
+# Basic Frontend - Spring 2021
+
+Lesson 7, Tuesday, 2021-03-23
+
+---
+
```

Comment:
you can add here an id for the topic, like: `<!-- .slide: id="functions-intro" -->` so that we can link it in the table of contents